QSB/Utility/FlagsHelper.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace QSB.Utility
+{
+    // Stolen from here : https://stackoverflow.com/questions/3261451/using-a-bitmask-in-c-sharp
+
+    public static class FlagsHelper
```

Comment:
What's this used for?

Reply:
using in playerregistry for changing states